<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ICT-5" failures="0" tests="2" name="rahulshettyacademy.tests.SubmitOrderTest" time="0.000" errors="1" timestamp="2025-01-06T13:42:52 IST" skipped="1">
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="submitOrder" time="0.000">
    <error message="java.io.FileNotFoundException: File &#039;E:\MavenProject\SeleniumFrameworkDesign\sr\test\java\rahulshettyacademy\data\PurchaseOrder.json&#039; does not exist" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: File 'E:\MavenProject\SeleniumFrameworkDesign\sr\test\java\rahulshettyacademy\data\PurchaseOrder.json' does not exist
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)
at org.testng.internal.Parameters.handleParameters(Parameters.java:803)
at org.testng.internal.Parameters.handleParameters(Parameters.java:728)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:911)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.io.FileNotFoundException: File 'E:\MavenProject\SeleniumFrameworkDesign\sr\test\java\rahulshettyacademy\data\PurchaseOrder.json' does not exist
at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:297)
at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1805)
at rahulshettyacademy.TestComponents.BaseTest.getJsonDataToMap(BaseTest.java:65)
at rahulshettyacademy.tests.SubmitOrderTest.getData(SubmitOrderTest.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)
... 30 more
]]>
    </error>
  </testcase> <!-- submitOrder -->
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="OrderHistoryTest" time="0.000">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- rahulshettyacademy.tests.SubmitOrderTest -->
